lia-app-uikit/ListTableViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ListTableViewController.swift
+//  lia-app-uikit
+//
+//  Created by Alice Wheeler on 2024-04-30.
+//
+
+import UIKit
+
+class ListTableViewController: UITableViewController {
```

Comment:
Actually you probably need to make the entrypoint of the app a UINavigationController which in turn has ListTableViewController as its root view

This is so that you can "push" the DetailViewController that Filip is making on top of the ListTableViewController